core/arch/arm/plat-d02/conf.mk

```diff
@@ -5,6 +5,7 @@ CFG_NUM_THREADS ?= 16
 CFG_CRYPTO_WITH_CE ?= y
 CFG_WITH_STACK_CANARIES ?= y
 CFG_CORE_STACK_PROTECTOR_STRONG ?= y
+CFG_CORE_FORTIFY_SOURCE ?= y
```

Comment:
This should be enabled in `mk/config.mk` instead, since we want it for all platforms by default.

Reply:
Have a fix for it, will push soon.